src/components/fold-media.css

```diff
@@ -0,0 +1,7 @@
+.FoldMedia .AttributionEditor-metadata {
+  top: -75px;
+}
```

Comment:
That's only the web app. Mobile integrations / smaller screens probably don't want the overlap. So, kill it?